docs/source/user-guide/kubernetes.md

```diff
@@ -33,3 +33,78 @@ docker build -t apache/datafusion-comet -f kube/Dockerfile .
 The exact syntax will vary depending on the Kubernetes distribution, but an example `spark-submit` command can be
 found [here](https://github.com/apache/datafusion-comet/tree/main/benchmarks).
 
+## Helm chart
+
+Install helm Spark operator for Kubernetes
+```bash
+helm repo add spark-operator https://kubeflow.github.io/spark-operator
+
+helm repo update
+
+helm install my-release spark-operator/spark-operator --namespace spark-operator --create-namespace --set webhook.enable=true
+````
+
+Check the operator is deployed
+```bash
+helm status --namespace spark-operator my-release
+
+NAME: my-release
+NAMESPACE: spark-operator
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+Create example Spark application file `spark-pi.yaml`
+```bash
+apiVersion: sparkoperator.k8s.io/v1beta2
+kind: SparkApplication
+metadata:
+  name: spark-pi
+  namespace: default
+spec:
+  type: Scala
+  mode: cluster
+  image: ghcr.io/apache/datafusion-comet:spark-3.4-scala-2.12-0.2.0-rc1
+  imagePullPolicy: IfNotPresent
+  mainClass: org.apache.spark.examples.SparkPi
+  mainApplicationFile: local:///opt/spark/examples/jars/spark-examples_2.12-3.4.2.jar
+  sparkConf:
+    "spark.executor.extraClassPath": "/opt/spark/jars/comet-spark-spark3.4_2.12-0.2.0-SNAPSHOT.jar"
+    "spark.driver.extraClassPath": "/opt/spark/jars/comet-spark-spark3.4_2.12-0.2.0-SNAPSHOT.jar"
```

Comment:
Do the jar files in the 0.2.0-rc1 image still have SNAPSHOT in the name? This should not be the case.

Reply:
fixed